packages/arb-ts/src/lib/rollupProtocol/rollupProtocol.ts

```diff
@@ -19,52 +19,16 @@
 import { Provider, BlockTag } from '@ethersproject/abstract-provider'
 import { BigNumber } from 'ethers'
 
-import { Whitelist__factory, Rollup__factory, Rollup } from '../abi'
+import { Rollup__factory } from '../abi'
 import { NodeConfirmedEvent, NodeCreatedEvent } from '../abi/Rollup'
 import { EventFetcher } from '../utils/eventFetcher'
-import { ADDRESS_ALIAS_OFFSET } from '../dataEntities/constants'
 
 /**
- * General information about the current network state
+ * General information about protocol level actions, WIP
  */
-export class NetworkState {
+export class RollupProtocol {
```

Comment:
I think this class has been deleted now, maybe we need to merge arb-ts-next into this branch

Reply:
ah

Reply:
updated